app/bundles/PageBundle/Translations/en_US/messages.ini

```diff
@@ -23,6 +23,11 @@ mautic.page.campaign.event.pagehit="Visits a page"
 mautic.page.campaign.event.pagehit_descr="Trigger actions on a page/url hit."
 mautic.page.campaign.event.devicehit="Device visit"
 mautic.page.campaign.event.devicehit_descr="Trigger device  on a page/url hit."
+mautic.page.campaign.condition.form.startdate="Start Date"
+mautic.page.campaign.condition.form.enddate="End Date"
+mautic.page.campaign.condition.form.timespent="Time spent (seconds)"
+mautic.page.campaign.condition.form.page="Page"
+mautic.page.campaign.condition.form.page.descr="Select the page this trigger applies to. If none are selected, it'll apply to any page."
```

Comment:
`mautic.page.campaign.condition.form.page.descr="Select the page this trigger applies to. If none are selected, it'll apply to any page."`

⏬ 

`mautic.page.campaign.condition.form.page.descr="Select the page this trigger applies to. If none is selected, it will be applied on all pages."`